src/vs-bicep/Bicep.VSLanguageServerClient/BicepLanguageServerClient.cs

```diff
@@ -74,7 +77,21 @@ public BicepLanguageServerClient(IProcessTracker processTracker)
 
             Debug.WriteLine($"Started {BicepContentTypeDefinition.ContentType} server with process ID {process.Process.Id}");
 
-            return new Connection(process.StandardOutput.BaseStream, process.StandardInput.BaseStream);
+            var connection = new Connection(process.StandardOutput.BaseStream, process.StandardInput.BaseStream);
+            var telemetryEvent = new TelemetryEvent("vs/bicep/clientInitialization");
+
+            if (connection is not null)
+            {
+                telemetryEvent.Properties["status"] = "succeeded";
+            }
+            else
+            {
+                telemetryEvent.Properties["status"] = "failed";
+            }
```

Comment:
remove dup code:

telemetryEvent.Properties["status"] = connection is null ? "failed" : "succeeded";

Reply:
Updated here:
https://github.com/Azure/bicep/commit/a2884898c13287b3eea95f3a37aef0790e3da8df

Thanks!